usaspending_api/etl/tests/integration/test_load_to_from_delta.py

```diff
@@ -676,6 +727,19 @@ def create_and_load_all_delta_tables(spark: SparkSession, s3_bucket: str, tables
     create_ref_temp_views(spark)
 
 
+def create_and_load_intermediate_delta_tables(spark: SparkSession, s3_bucket: str):
```

Comment:
Should be able to reuse `create_and_load_all_delta_tables` since it is going to load in tables in the order you specify and prioritize the `load_table_to_delta` command before the `load_query_to_delta`.